packages/meditrak-server/src/routes/GETMapOverlays/assertMapOverlaysPermissions.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import flattenDeep from 'lodash.flattendeep';
+import keyBy from 'lodash.keyby';
+
+import { hasBESAdminAccess } from '../../permissions';
+
+export const filterMapOverlaysByPermissions = async (accessPolicy, models, mapOverlays) => {
```

Comment:
I haven't looked closely at this one, but will re-review this after you've addressed the other comments in the PR. I have a broader comment on map overlays though: the fact that they have `countryCodes` implies that the entity won't be project, so I believe it can be simplified considerably? Also worth noting that if it does need the project logic, you'd be better to import and use the util for checking visualisations you made.